locales/en-US/app.ftl

```diff
@@ -30,6 +30,18 @@ Home--load-from-url-submit-button =
     .value = Load
 
 Home--documentation-button = Documentation
+Home--menu-button = Enable Profiler Menu Button
+Home--menu-button-instructions = 
```

Comment:
nit: there is a trailing space after the `=`.
note: also you can make your IDE/text editor to automatically remove the trailing spaces. Then you won't have to think about it.